Makefile

```diff
@@ -131,6 +131,7 @@ server-makemigrations:
 
 server-migrate:
 	$(PYTHON) quipucords/manage.py migrate --settings quipucords.settings -v 3
+	$(PYTHON) quipucords/manage.py randomize_db_sequences
```

Comment:
I'd change put this in a separate `make role` because production does depend on `make server-migrate`

https://github.com/quipucords/quipucords/blob/61610c14c5d757b59569c34624ced23d16adad8c/deploy/entrypoint_web.sh#L15

Reply:
and the reason I insist this to be a separate command is because we know that randomizing db sequences will horribly bork the UI

Reply:
Fair enough. The UI is already horribly broken, but I guess we shouldn't make it _obvious_ that it's already horribly broken. 😕 

FWIW, it seems trivially easy to encounter this ugly UI ID bug _without_ randomizing the database sequences. I've been hitting it somewhat regularly since I joined the project many months ago.

Reply:
Done!

It's `make server-randomize-sequences` to try to follow the current `server-` naming convention.